### ISSUE TITLE
the Chinese guide 'How to Build ClickHouse on Mac OS X'  is outdate 

### ISSUE THREAD

**Issue by liuzhizu** [ClickHouse/ClickHouse#12051]
the Chinese guide 'How to Build ClickHouse on Mac OS X'  is outdate  ,the link is https://clickhouse.tech/docs/zh/development/build-osx/  ,which Build ClickHouse is outdate ：

```
mkdir build
cd build
cmake .. -DCMAKE_CXX_COMPILER=`which g++-8` -DCMAKE_C_COMPILER=`which gcc-8`
ninja
cd ..
```

**Comment by liuzhizu** [comment]
the outdate guide can result in build fail,please attention!

**Comment by filimonov** [comment]
Can you send a PR with fix? There is an edit button on a page (looks like pencil)

**Comment by BohuTANG** [comment]
Fix #12071
